modules/oadp-s3-compatible-backup-storage-providers.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_content-type: CONCEPT
+[id="oadp-s3-compatible-backup-storage-providers_{context}"]
+= AWS S3 compatible backup storage providers
+
+OADP supports a variety of object storage providers for different backup and snapshot operations.
+
+== AWS S3 drivers for backup storage locations
+
+=== Supported
+
+The following AWS S3 compatible object storage providers, are fully supported by OADP through the AWS plugin, for use as backup storage locations:
+
+* MinIO
+* Multicloud Object Gateway (MCG) with NooBaa
+* Amazon Web Services (AWS) S3
+
+[NOTE]
+====
+The following compatible object storage providers are supported and have their own Velero object store plugins:
+
+* Google Cloud Platform (GCP)
+* Microsoft Azure
+====
+
+=== Unsupported
+
+The following AWS S3 compatible object storage providers, are known to work with Velero through the AWS plugin, for use as backup storage locations, but are unsupported and have not been tested by Red Hat:
+
+* IBM Cloud
+* Oracle Cloud
+* DigitalOcean
+* NooBaa
+* Tencent Cloud
+* Ceph RADOS v12.2.7
+* Quobyte
+* Cloudian HyperStore
+
+=== Known issues
```

Comment:
This content seems like it either needs to be moved to the RNs or relabeled as IMPORTANT with more of explanation as to the consequences of using them.